src/components/TextInputWithInfoLink.tsx

```diff
@@ -0,0 +1,45 @@
+import { ChangeEvent, FC, InputHTMLAttributes, useState } from 'react';
+import { InfoCircleWithTooltip } from './InfoCircleWithTooltip';
+
+interface TextInputWithInfoLinkProps extends InputHTMLAttributes<HTMLInputElement> {
```

Comment:
Any reason why some of the default props are here? I think we can get rid of any non-custom ones: `placeholder`, `name`, `value` and `onChange`.